python-client/giskard/ml_worker/core/model_explanation.py

```diff
@@ -64,13 +64,41 @@ def explain_text(model: GiskardModel, input_df: pd.DataFrame,
         model.prediction_function, input_df, text_column
     )
     try:
-        text_explainer.fit(text_document, prediction_function)
-        return text_explainer.show_prediction(target_names=model.classification_labels)
+        for i in range(10):
```

Comment:
could you extract 10 into a constant with a self explanatory name like `TEXT_EXPLAIN_ATTEMPTS` ?